releasenotes/notes/support_excitedstatessolver_bopes-2bbb5a282b1ff582.yaml

```diff
@@ -0,0 +1,18 @@
+features:
+  - |
+    The solver of BOPESSampler now accepts solvers of type
+    `Union[GroundStateSolver,ExcitedStatesSolver]` instead of only a `GroundStateSolver`. 
+    This generalizes to excited states the sampling 
+    of nuclear coordinates for Born Oppenheimer Potential Energy Surfaces.
+    Adds the `get_qubit_operators()` in
+    :class:`qiskit_nature.algorithms.ExcitedStatesEigensolver`
+    This matches the format of
+    :class:`qiskit_nature.algorithms.GroundStatesEigensolver`
+    BOPESSampler now also accepts auxiliary operators to pass to the solver.
+    Geometry-dependent observable can also be defined as auxiliaries
+
+upgrade:
+  - |
```

Comment:
I don't think we ever really use this section. I would just append this information at the end of the `features` section

Reply:
Done